### PR TITLE
Fix default plural

### PR DESCRIPTION
**Pull request by thefunny42**

### CODE REVIEW

File: src/zope/i18n/tests/test_translationdomain.py [hunk 1]
```diff
@@ -115,13 +115,15 @@ def testMessageIDRecursiveTranslate(self):
...
-        # we should return a defualt translation.
+        self.assertEqual(
+            "Color: BLUE/YELLOW",
+            translate(u"Color: ${color1}/${color2}", mapping=mapping,
+                      target_language='en'))
+
+        # If we have mapping with a message id from a different
+        # domain, make sure we use that domain, not ours. If the
+        # message domain is not registered yet, we should return a
+        # defualt translation.
```
Comment (mgedmin) [inline src/zope/i18n/tests/test_translationdomain.py]: Typo: defualt -> default.

(Yeah, it's an old typo, but you touched the code last ;)

(Also, a few lines above there's a "Recusrive" typo.)

**Comment by thefunny42** [comment]
Can I get a release please?

**Comment by jamadden** [comment]
4.6.0 has been pushed to PyPI.

**Comment by mgedmin** [comment]
I would've called it 4.5.1, given it has only one bugfix after 4.5.0.

(That's the danger of using short X.Y instead of X.Y.0 versions -- zest.releaser bumps it too much in postrelease.)

**Comment by jamadden** [comment]
I wasn't entirely clear to me that it was *just* a bugfix and not new behaviour, so I went with the minor bump.

**Comment by thefunny42** [comment]
Thanks!

**Pull request merged**